code/renderers/svelte/src/components/PreviewRender.svelte

```diff
@@ -53,4 +51,4 @@
   }
 </script>
 
-<SlotDecorator {Component} {props} {on} {argTypes} />
+<SlotDecorator {Component} props={props} {on} {argTypes} />
```

Comment:
this change is not necessary
```suggestion
<SlotDecorator {Component} {props} {on} {argTypes} />
```